src/main.rs

```diff
@@ -249,20 +249,13 @@ fn main() {
     let args = Cli::parse();
 
     // Determine the usage of styles
-    #[cfg(target_os = "windows")]
-    let ansi_support = yansi::Paint::enable_windows_ascii();
```

Comment:
I suppose by https://docs.rs/yansi/1.0.1/yansi/index.html#windows we can omit this. Fair enough

Reply:
Yeah the commit message explained ;)

Reply:
Ah, I didn't see that, I'll make sure to include it in the merged commit